.github/workflows/on-pull-request.yml

```diff
@@ -17,31 +17,31 @@ jobs:
             - name: Checkout source
               uses: actions/checkout@v4
 
-            - name: Build web with PreTeXt
+            - name: Build student web version with PreTeXt
               uses: siefkenj/pretext-build-action@main
               with:
-                  pretext-command: build web
+                  pretext-command: build web-student
                   project-root: "."
                   output-dir: "."
 
-            - name: Build print with PreTeXt
+            - name: Build student print version with PreTeXt
               uses: siefkenj/pretext-build-action@main
               with:
-                  pretext-command: build print
+                  pretext-command: build print-student
                   project-root: "."
                   output-dir: "."
 
-            - name: Build instructor version with PreTeXt
+            - name: Build instructor web version with PreTeXt
               uses: siefkenj/pretext-build-action@main
               with:
-                  pretext-command: build instructor
+                  pretext-command: build web

```

Comment:
Clever, this didn't occur to me.  I was trying to figure out how to make Codechat run 'pretext build instructor' instead.